tests/test_project.py

```diff
@@ -49,13 +49,3 @@ def test_schema_load_validation(self):
     def test_target_by_value(self):
         target = Target(Target.PULL_REQUEST)
         assert target == Target.PULL_REQUEST
-
-    def test_placeholder_replacement(self):
```

Comment:
You don't think we need a test case like this?

Reply:
It has become part of the general chart composition logic now and with that it's tested by the chart roundtrip tests https://github.com/Vandebron/mpyl/pull/88/files#diff-929bc8ca4b62bd7c9555d2c69442c95034a1de4e0767816625849eed8da9d1d2L18